ci/ios-e2e-template.yml

```diff
@@ -0,0 +1,44 @@
+# https://docs.microsoft.com/azure/devops/pipelines/languages/android
+parameters:
+    script: 'npx mocha --timeout 480000 build/test/functional/ -g @skip-ci -i --exit'
+    name: ios-e2e-test
+    dependsOn: ''
+    iosVersion: 12.1
+    xcodeVersion: 10.1
+    deviceName: 'iPhone X'
+
+jobs:
+  - job: ${{ parameters.name }}
+    variables:
+      PLATFORM_VERSION: ${{ parameters.iosVersion }}
+      DEVICE_NAME: ${{ parameters.deviceName }}
+      MOCHA_FILE: '${{ parameters.name }}-tests.xml'
+      CI: true
+    pool:
+      vmImage: 'macOS 10.13'
+    steps:
+    - checkout: self
+      submodules: true
+    - script: |
+        ls /Applications/
+        sudo xcode-select -s /Applications/Xcode_${{ parameters.xcodeVersion }}.app/Contents/Developer
```

Comment:
[nits]
You can also set `DEVELOPER_DIR=/Applications/Xcode_${{ parameters.xcodeVersion }}.app/Contents/Developer` as an environment variable insetad of calling `sudo xcode-select -s`. I think this works on Azure.